tests/testnet-performance/nightly.yml

```diff
@@ -0,0 +1,77 @@
+steps:
+  - command: "tests/testnet-performance/testnet-automation.sh"
+    label: "Run performance testnets on EDGE channel CPU ONLY"
+#    branches: 'master'
```

Comment:
`branches` are commented out while I test this in PR-land, otherwise the buildkite job would filter out everything here until this code actually lives on said branch.